### DISCUSSION TITLE
[Feature] <Auto select duplicate based on prefrence>

### DISCUSSION THREAD

**Discussion by JakobEdvardsson** [immich-app/immich#10163] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Auto select duplicate based on preferences. I have a lot of duplicate images, some have no metadata and some have. For example, I have about 2000 images from one day (day I started using Immich) most of these are duplicate and have wrong dates. I wish there was a way to auto select everything before this date.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#10665
